.travis.yml

```diff
@@ -1,23 +1,29 @@
 language: python
 sudo: false
-python:
-  - 2.6
-  - 2.7
-  - 3.2
-  - 3.3
-  - 3.4
-  - 3.5
-  - nightly
-  - pypy
-  - pypy3
 install:
-  - pip install -e .
+  - pip install tox
   - pip list
-script:
-  - python pycodestyle.py --testsuite testsuite
-  - python pycodestyle.py --statistics pycodestyle.py
-  - python pycodestyle.py --doctest
-  - python setup.py test
+script: tox
+matrix:
```

Comment:
@IanLee1521 it matches the usage of the majority of projects that have test environments that are more than just "test on these versions of python". It would allow the project to, for example, also run a tox test environment to ensure that the documentation passes a tool like doc8 and builds in sphinx with all warnings treated as errors, etc. (Effectively testing that your docs meet a certain style and actually build.)

Given how many more projects use the matrix approach rather than using the rather young tox-travis tool (which I've not seen any project of any size use), I think this is going to be the better way to manage our usage of tox within travis.